### PR TITLE
Add stubs to all local modules

### PR DESCRIPTION
**Pull request by SPPearce**
The download test is currently failing because it is supposed to be only running in stub mode, but the local modules didn't have stubs.

### CODE REVIEW

**Review by znorgaard — APPROVED** [review]
LGTM! Also ran the download action manually for this branch, and it passes: https://github.com/nf-core/fastquorum/actions/runs/10724370756/job/29739862127

**Comment by SPPearce** [comment]
Hmm, why did the tests fail here though? Is it because I was accidentally against master the first time?

File: modules/local/fgbio/collectduplexseqmetrics/main.nf [hunk 1]
```diff
@@ -37,6 +37,21 @@ process FGBIO_COLLECTDUPLEXSEQMETRICS {
         --duplex-umi-counts=true \\
         $args;
 
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        fgbio: \$( echo \$(fgbio --version 2>&1 | tr -d '[:cntrl:]' ) | sed -e 's/^.*Version: //;s/\\[.*\$//')
+    END_VERSIONS
+    """
+    stub:
```
Comment (SPPearce) [inline modules/local/fgbio/collectduplexseqmetrics/main.nf:45]:
```suggestion

    stub:
```

**Comment by znorgaard** [comment]
> Hmm, why did the tests fail here though? Is it because I was accidentally against master the first time?

Yes. That's why I did the explicit run of the action manually while specifying this branch. Otherwise it's hardcoded to use `dev` when not triggered as a dispatch that includes inputs. https://github.com/nf-core/fastquorum/blob/ea307015cf134358883d48fcd7a86b365a0a8bb2/.github/workflows/download_pipeline.yml#L50

You can see it pulled `dev` in the "Download the pipeline" step as the "pipeline revision" for the original trigger from targeting `main`. https://github.com/nf-core/fastquorum/actions/runs/10714623881/job/29708611287

And pulled `add_stubs` for the one I manually triggered. https://github.com/nf-core/fastquorum/actions/runs/10724370756/job/29739862127

**Pull request merged**